chalicelib/checks/wfr_checks.py

```diff
@@ -1,1448 +1,1456 @@
+        meta_workflow_run_uuids
+    )
+    check.summary = msg
     check.brief_output.append(msg)
-    check.full_output['metawfrs_to_run'] = {'titles': metawfr_titles, 'uuids': metawfr_uuids}
+    check.status = "WARN"
+    check.full_output["metawfrs_to_run"] = {
+        "titles": meta_workflow_run_titles,
+        "uuids": meta_workflow_run_uuids,
+    }
+    if not meta_workflow_run_uuids:
+        check.allow_action = False
     return check
 
 
 @action_function()
 def line_count_test(connection, **kwargs):
     start = datetime.utcnow()
-    action = ActionResult(connection, 'line_count_test')
-    action_logs = {'metawfrs_that_passed_linecount_test': [], 'metawfrs_that_failed_linecount_test': []}
+    action = ActionResult(connection, "line_count_test")
+    action_logs = {
+        "metawfrs_that_passed_linecount_test": [],
+        "metawfrs_that_failed_linecount_test": [],
+    }
     my_auth = connection.ff_keys
-    env = connection.ff_env
-    check_result = action.get_associated_check_result(kwargs).get('full_output', {})
-    action_logs['check_output'] = check_result
-    action_logs['error'] = []
-    metawfr_uuids = check_result.get('metawfrs_to_run', {}).get('uuids', [])
+    check_result = action.get_associated_check_result(kwargs).get("full_output", {})
+    action_logs["check_output"] = check_result
+    action_logs["error"] = []
+    metawfr_uuids = check_result.get("metawfrs_to_run", {}).get("uuids", [])
     for metawfr_uuid in metawfr_uuids:
         now = datetime.utcnow()
-        if (now-start).seconds > lambda_limit:
-            action.description = 'Did not complete action due to time limitations'
+        if (now - start).seconds > lambda_limit:
+            action.description = "Did not complete action due to time limitations"
             break
         try:
-            metawfr_meta = ff_utils.get_metadata(metawfr_uuid, add_on='?frame=raw', key=my_auth)
+            metawfr_meta = ff_utils.get_metadata(
+                metawfr_uuid, add_on="frame=raw", key=my_auth
+            )
             # we have a few different dictionaries of steps to check output from in linecount_dicts.py
             # the proband-only and family workflows have the same steps, so we assign the proband_SNV_dict
-            if 'Proband-only' in metawfr_meta['title'] or 'Family' in metawfr_meta['title']:
+            if (
+                "Proband-only" in metawfr_meta["title"]
+                or "Family" in metawfr_meta["title"]
+            ):
                 steps_dict = proband_SNV_dict
             # trio has novoCaller, so it has a separate dictionary of steps
-            elif 'Trio' in metawfr_meta['title']:
+            elif "Trio" in metawfr_meta["title"]:
                 steps_dict = trio_SNV_dict
             # cnv/sv is a completely different pipeline, so has a many different steps
-            elif 'CNV' in metawfr_meta['title']:
+            elif "CNV" in metawfr_meta["title"]:
```

Comment:
It might be more robust, rather than relying on the title being formatted a certain way, to make explicit properties for these in MWF that get embedded into MWFR and then supplied into the check output. This would require schema changes, so I think it's fine to postpone this, but in the long term probably better to not rely on the title being manually formatted one specific way.